setup.py

```diff
@@ -47,15 +47,18 @@
                   'tomas.angles@ens.fr, gabriel.huang@umontreal.ca, roberto.leonarduzzi@ens.fr'),
     url=URL,
     download_url='https://github.com/kymatio/kymatio/releases',
+    project_urls={
+        'Documentation': 'https://www.kymat.io/codereference.html',
+        'Source': 'https://github.com/kymatio/kymatio/',
+        'Tracker': 'https://github.com/kymatio/kymatio/issues',
+        'Authors': 'https://github.com/kymatio/kymatio/blob/master/AUTHORS.md'
```

Comment:
Is this a link for humans or machines? If for machines, maybe raw link is better. If for humans, this link is better

Reply:
humen AFAIK